app/javascript/article-form/elements/imageManagement.jsx

```diff
@@ -178,8 +193,7 @@ export default class ImageManagement extends Component {
               <em>
                 To add a cover image for the post, add &nbsp;
                 <code>cover_image: direct_url_to_image.jpg</code>
-&nbsp; to the
-                frontmatter
+                &nbsp; to the frontmatter
```

Comment:
I have no idea what this change is, probably the prettify or my editor. I'll revert it if necessary.

Reply:
Wherever it came from, it looks like a sensible change, let's keep it.